### PR TITLE
Update for Objectify 6.0.4

### PR DESCRIPTION
**Pull request by briandealwis**
[Objectify 6 requires Java 8](https://github.com/objectify/objectify/blob/master/pom.xml#L86) so also add `javaVersion="1.8"` to the library definition.

### CODE REVIEW

File: plugins/com.google.cloud.tools.eclipse.appengine.libraries/plugin.xml [hunk 1]
```diff
@@ -65,6 +66,7 @@
           group="non-appengine-standard"
           name="Objectify"
           tooltip="%objectify.tooltip"
+          javaVersion="1.8"
```
Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.libraries/plugin.xml]: why was this needed?

Comment (briandealwis) [reply plugins/com.google.cloud.tools.eclipse.appengine.libraries/plugin.xml]: While looking at why the test failed, I noticed the`Library` instance for Objectify had`javaVersion` set to `"1.7"`. Objectify 6 requires Java 8.
